docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -131,13 +129,13 @@ kubectl auth can-i \
 
 ```sh
 curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-namespaced-admin.yaml \
-  | sed "s/namespace: <SA_NAMESPACE>/namespace: <namespace>"/g \
+  | sed "s/namespace: <serviceaccount_namespace>/namespace: <namespace>"/g \
```

Comment:
`platform-namespaced-admin.yaml` has a placeholder <SA_NAMESPACE>. So this updated sed command does not replace the placeholder anymore.

Either

1. Change the command to use the correct placeholder
2. Change the `platform-namespaced-admin.yaml` to use the correct placeholder

Reply:
Changed back to <SA_NAMESPACE>. 
The change will be delivered via a new PR.